app/views/partners/index.html.erb

```diff
@@ -28,7 +28,8 @@
                 <% end %>
               </div>
               <hr>
-                <%= f.button :submit , 'Filter' %>
+                  <a href="/partners" class="btn">Reset</a>
```

Comment:
Minor things: would it be possible to not show the reset button if nothing is selected?

And there is a small discrepancy in scale of the two buttons side by side. 

![Screenshot_2023-04-27_16-16-50](https://user-images.githubusercontent.com/109223824/234907989-c9c56506-3747-4239-9f14-a6ac2bbaf89f.jpg)